src/main/java/com/shepherdjerred/capstone/common/lobby/PlayerSlots.java

```diff
@@ -70,9 +70,6 @@ PlayerSlots setPlayer(QuoridorPlayer playerId, Player player) {
 
   PlayerSlots addPlayer(Player player) {
     checkNotNull(player);
-
-    var newQuoridorPlayerMap = new HashMap<>(playerIdMap);
-    newQuoridorPlayerMap.put(QuoridorPlayer.fromInt(playerIdMap.size() + 1), player);
-    return new PlayerSlots(newQuoridorPlayerMap, playerCount);
+    return setPlayer(QuoridorPlayer.fromInt(playerIdMap.size() + 1), player);
```

Comment:
This won't always work. For example, if it is a two player game and one player is in the lobby in player two's slot.